### PR TITLE
Lizards are no longer afraid of themselves

### PR DESCRIPTION
**Pull request by super12pl**
## About The Pull Request
Tinfoil hat no longer makes wearer scared of lizards when worn by a lizard.

## Why It's Good For The Game
Fix good

## Changelog
:cl:
fix: Lizards are no longer afraid of themselves
/:cl:

### CODE REVIEW

**Comment by Kryyto** [comment]
Make sense

**Comment by zeskorion** [comment]
My phobia rework pr already does this

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -305,17 +305,23 @@
...
+			QDEL_NULL(paranoia_lizard)
+		paranoia_lizard = new()
+		paranoia_lizard.clonable = FALSE
+		user.gain_trauma(paranoia_lizard, TRAUMA_RESILIENCE_MAGIC)
+		to_chat(user, "<span class='warning'>As you don the foiled hat, an entire world of conspiracy theories and seemingly insane ideas suddenly rush into your mind. What you once thought unbelievable suddenly seems.. undeniable. Everything is connected and nothing happens just by accident. You know too much and now they're out to get you. </span>")
+	else
 		if(paranoia)
 			QDEL_NULL(paranoia)
 		paranoia = new()
 		paranoia.clonable = FALSE
```
Comment (PowerfulBacon) [inline code/modules/clothing/head/misc_special.dm:324]: This whole section has a lot of repeated code, I feel like if conspirisies without the lizards was a subtype of conspiricies you could make the whole thing a lot less and just have
```
if(paranoia)
		QDEL_NULL(paranoia)
if(is_species(/datum/species/lizard))
		paranoia = new /datum/brain_trauma/mild/phobia/conspiracies/nolizards()
else
		paranoia = new ()
paranoia_lizard.clonable = FALSE
		user.gain_trauma(paranoia_lizard, TRAUMA_RESILIENCE_MAGIC)
		to_chat(user, "<span class='warning'>As you don the foiled hat, an entire world of conspiracy theories and seemingly insane ideas suddenly rush into your mind. What you once thought unbelievable suddenly seems.. undeniable. Everything is connected and nothing happens just by accident. You know too much and now they're out to get you. </span>")
```

**Comment by zeskorion** [comment]
closing in favor of https://github.com/BeeStation/BeeStation-Hornet/pull/2129

**Pull request closed without merging**